python/paddle/fluid/transpiler/inference_transpiler.py

```diff
@@ -65,8 +65,43 @@ def transpile(self, program, place, scope=None):
         if use_mkldnn:
             self._fuse_conv_bias_mkldnn(program)
             self._fuse_conv_relu_mkldnn(program)
+            self._fuse_conv_eltwise_mkldnn(program)
+            self._fuse_conv_relu_mkldnn(
+                program)  # ResNet residual block merging
             self._fuse_bn_relu_mkldnn(program)
 
+    def _fuse_conv_eltwise_mkldnn(self, program):
```

Comment:
Please transform this to an IR fuse pass.

The inference transpiler has been abandoned and no need to update them, and we will delete these files in several weeks.

You can try them for research locally, but no need to update these files.

Reply:
@Superjomn, @luotao1  We are actively working also on pass mechanism (https://github.com/PaddlePaddle/Paddle/pull/13368). We don't mind removing transpiler code in several weeks. But please merge this PR in current form as:
-we need C++ integration part to continue work on pass mechanism (it is hard to be dependent on internal branch, which is not rebased on develop branch on daily basis)
-our automated tests rely on public code. If you don't merge the transpiler part, we won't be able to pass any performance numbers to you in upcoming weeks

Reply:
Thank you for the suggestion @Superjomn. The purpose of this transpiler was to present the performance boost the fuse provides and have a way of testing whether it works. The IR fuse pass implementation has already been prepared by @tpatejko but he is just waiting for this PR to be merged.

Reply:
I see, hoping for the corresponding IR passes.